modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/common/B2BRESTTestBase.java

```diff
@@ -0,0 +1,222 @@
+import org.hamcrest.Matcher;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.identity.integration.common.clients.usermgt.remote.RemoteUserStoreManagerServiceClient;
+
+import java.rmi.RemoteException;
+import java.util.ResourceBundle;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.StringContains.containsString;
+
+/**
+ * Base test class for B2B organization management REST API tests.
+ */
+public class B2BRESTTestBase extends RESTTestBase {
+
+    protected static final String ORGANIZATION_CONTEXT_IN_URL = "/o/%s";
+    protected static final String SERVICES = "/services";
+    private static ResourceBundle errorProperties = ResourceBundle.getBundle("RESTAPIErrors");
+
+    protected String authenticatingUserName;
+    protected String authenticatingCredential;
+    protected String tenant;
+    protected AutomationContext context;
+
+    protected RemoteUserStoreManagerServiceClient remoteUSMServiceClient;
+    protected String swaggerDefinition;
+
+    protected String basePath = StringUtils.EMPTY;
+
+    /**
+     * Initialize the REST API validation requirements configuring the OpenApiValidationFilter
+     *
+     * @param swaggerDefinition Swagger definition name.
+     * @param basePathInSwagger Basepath that is defined in the swagger definition (ex: /api/users/v1).
+     * @param basePath Basepath of the current test run (ex: /o/{organization-domain}/api/users/v1).
+     * @throws RemoteException Throws this exception.
+     */
+    protected void init(String swaggerDefinition, String basePathInSwagger, String basePath) throws RemoteException {
+
+        super.init(swaggerDefinition, basePathInSwagger, basePath);
+        this.basePath = basePath;
+    }
+
+    /**
+     * Invoke given endpointUri for GET with Basic authentication, authentication credential being the
+     * authenticatingUserName and authenticatingCredential.
+     *
+     * @param endpointUri Endpoint to be invoked.
+     * @return response of get request.
+     */
+    protected Response getResponseOfGet(String endpointUri) {
+
+        return given().auth().preemptive().basic(authenticatingUserName, authenticatingCredential)
+                .contentType(ContentType.JSON)
+                .header(HttpHeaders.ACCEPT, ContentType.JSON)
+                .log().ifValidationFails()
+                .when()
+                .get(endpointUri);
+    }
+
+    /**
+     * Invoke given endpointUri for POST with given body and Basic authentication, authentication credential being the
+     * authenticatingUserName and authenticatingCredential.
+     *
+     * @param endpointUri Endpoint to be invoked.
+     * @param body        Payload.
+     * @return Response of post request.
+     */
+    protected Response getResponseOfPost(String endpointUri, String body) {
+        return given().auth().preemptive().basic(authenticatingUserName, authenticatingCredential)
```

Comment:
missing new line after the method signature

Reply:
Fixed by - https://github.com/wso2/product-is/pull/16566/commits/177717a2f7457f7a3e486d6359b440b160ab1fe5